### PR TITLE
SQL queries

### PR DESCRIPTION
**Pull request by touredri**
In this PR I create a table and insert data inside it.
bellow the screenshot for all queries.
Capture d’écran 2023-06-06 014121

Capture d’écran 2023-06-06 014201
Capture d’écran 2023-06-06 014232
Capture d’écran 2023-06-06 014357
Capture d’écran 2023-06-06 014532
Capture d’écran 2023-06-06 014849

Capture d’écran 2023-06-06 012152
Capture d’écran 2023-06-06 014042

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+insert into animals values (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+insert into animals values (2, 'Gabumon', '2018-11-15', 2, true, 8.0);
+insert into animals values (3, 'Pikachu', '2021-01-7', 1, false, 15.04);
+insert into animals values (4, 'Devimon', '2017-05-12', 5, true, 11.0);
```
Comment (bobb-Rob) [inline data.sql:5]: - [ ] I notice that you have not added code that addresses the questions below: 

- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 
**Kindly add these codes and attach screenshots of the results of your queries.**
Please label each screenshot with the question. I see that you added some screenshots, please add labels.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @touredri, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (touredri) [reply data.sql:5]: @bobb-Rob please these codes are inside the queries.sql file
THANKS

Comment (touredri) [reply data.sql:5]: And the screenshots are on the pull request description

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @touredri  👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion with complaints about the last reviewer @bobb-Rob . Below are my findings:

You seem to have made commits directly to your base branch before creating the branch called "createTable". In your base file, the queries that you attached the screenshots for are there, however, in this branch (a feature branch which is to go through review) the queries are not there. Below is how the queries.sql file looks like right now:

image

**Note:** above I checked to the commit that was there before the CR (@bobb-Rob ) made the review to have an understanding of how the code base looked like before he made a review.

**What you should do:**

You need to ensure that the queries are available in this branch. Ideally you need to copy the queries and push them as a commit. This way, they will appear in the "files changed" and the reviewers will be able to review your code as per the changes that are in this branch.

**Kindly note that the reviewers will only review code that is in this branch and not code from other branches.**

:warning: Remember the issue here is that the queries are not visible in this branch. :warning:
 
In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by touredri** [comment]
@Nemwel-Boniface 
and for screenshots?

**Review by Nemwel-Boniface — COMMENTED** [review]
@touredri The screenshots are there. No need to change anything. 

Please feel free to request another review now. :rocket:

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @touredri, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Animal table created with all necessary data✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**